docs/explanation/bgp-redirect.rst

```diff
@@ -0,0 +1,158 @@
+========================
+OVN integration with BGP
+========================
+
+OVN fully enabled support for integration with BGP in version ``25.03``. The
+support includes the ability to redirect BGP control-plane traffic to a
+specific ``Logical Switch Port`` and the ability of the ``ovn-controller`` to
+advertise and learn routes via a `Linux VRF`_.
+
+BGP control plane
+-----------------
+
+To avoid implementing the BGP protocol inside the OVN, the upstream project
+decided to provide a way to redirect the BGP control plane traffic from a
+``Logical Router Port`` to a ``Logical Switch Port``. As a result, an
+off-the-shelf BGP daemon can be run inside a VRF, bound to the ``Logical Switch
+Port``. Routing information is confined inside the VRF, leaving the routing
+table of the main host unaffected. And finally, the BGP daemon appears to the
+rest of the network as if it was bound to the ``Logical Router Port``.
+
+The last point is important for the purpose of hardware offload. If the BGP
+daemon acts as if it was bound to the ``Logical Router Port``, it also
+advertises its routes with the next hop address of the ``Logical Router Port``.
+The data-plane traffic can be then accelerated via the hardware offload without
+any further intervention.
+
+Route advertisement and learning via VRF
+----------------------------------------
+
+The ``ovn-controller`` is capable of maintaining VRFs and using them to learn
+and advertise routes.
+
+When the chassis binds a ``Logical Router Port`` configured to advertise or
+learn routes, it creates a VRF and inserts the routes to the VRF table. A
+BGP daemon can be configured to use the OVN's VRF table, and announce the
+routes to its peers. Conversely, when the BGP daemon learns routes from its
+peers, it inserts them to the VRF, from which they are picked up by the
+``ovn-controller`` and learned by the OVN.
+
+What MicroOVN sets up
+---------------------
+
+MicroOVN can simplify the BGP integration setup described in the previous
+sections. For more information on how to do it, see:
+:doc:`How-To: Configure OVN BGP integration </how-to/bgp>`
+
+To fully set up BGP redirecting, MicroOVN requires following:
+
+* one or more physical interfaces that provide connectivity to the external
+  networks
+* VRF table ID that the OVN will create and to which the internal routes will
+  be leaked.
+* (Optional) AS number that will be used by the BGP daemon to identify itself.
+  Note that if this is not provided, the BGP daemon won't be configured. You
+  can choose to omit the ASN if you wish to configure the BGP daemon manually.
+
+With these information provided, MicroOVN will then set up an OVN ``Logical
```

Comment:
Either change `these` to `this` or shoot in "pieces" between `these` and `information`